modules/installation-osp-enabling-swift.adoc

```diff
@@ -35,3 +35,8 @@ $ openstack role add --user <user> --project <project> swiftoperator
 ----
 
 Your {rh-openstack} deployment can now use Swift for the image registry.
+
+[NOTE]
+====
+OpenStack with Ceph storage backend does not support `.qcow2` image format. You must store CoreOS image in `.raw` format. You must follow link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.2/html/creating_and_managing_images/ch-image-service#section-convert-image[the manual process] to transform the image from `.qcow2` to `.raw`.
```

Comment:
```suggestion
{rh-openstack} with the Ceph storage back end does not support the `.qcow2` image format. You must store the CoreOS image in the `.raw` format. You must follow link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.2/html/creating_and_managing_images/ch-image-service#section-convert-image[the manual process] to transform the image from `.qcow2` to `.raw`.
```

Would it be correct to rewrite this to state that RHOSP + Ceph _only_ supports RAW? Would result in the limitation being described with positive language instead of negative.

Reply:
Addressed